internal/app/helm_helpers.go

```diff
@@ -93,15 +94,25 @@ func checkHelmVersion(constraint string) bool {
 // helmPluginExists returns true if the plugin is present in the environment and false otherwise.
 // It takes as input the plugin's name to check if it is recognizable or not. e.g. diff
 func helmPluginExists(plugin string) bool {
-	cmd := helmCmd([]string{"plugin", "list"}, "Validating that [ "+plugin+" ] is installed")
-
-	result := cmd.Exec()
-
-	if result.code != 0 {
-		return false
+	if plugins == nil {
```

Comment:
This will only ever run once so if we check for more than one plugin, only the first one will be checked for and the remaining ones will  always return false.

Reply:
It's filling the global map with all results from `helm plugin list` except the "NAME" header

Reply:
yes, you're right, I guess the `plugin` threw me off.